src/cryptsetup/cryptsetup.c

```diff
@@ -199,6 +201,10 @@ static int parse_one_option(const char *option) {
                 arg_verify = true;
         else if (STR_IN_SET(option, "allow-discards", "discard"))
                 arg_discards = true;
+        else if (streq(option, "same_cpu_crypt"))
+                arg_same_cpu_crypt = true;
+        else if (streq(option, "submit_from_crypt_cpus"))
+                arg_submit_from_crypt_cpus = true;
```

Comment:
hmm, all other options we use here (much like the options in fstab) use dashes rather underscores for separating words. I think these ones should do too. (or maybe we should do it like on the kernel cmdline, and consider _ and - the same, i.e. generalize the code in src/basic/proc-cmdline.c a bit, and maybe add streq_dash_underscore_same() or so which is like streq() but considers dash and underscore the same?

Reply:
Indeed...

Reply:
Hmm... Complicated...